lib/tasks/generate_elo_csv.rake

```diff
@@ -143,8 +127,8 @@ task generate_elo_csv: :environment do
 
     ts_min, ts_max = get_min_max_mu(@players_by_name.values)
 
-    CSV.open("elo_output_decay.csv", "w") do |csv|
-      csv << ["name", "games played", "wins", "losses", "last_played", "ts_elo", "ts_mu", "ts_sigma"]
+    CSV.open("elo_output.csv", "w") do |csv|
+      csv << ["name", "games played", "wins", "losses", "last_played", "ts_elo", "ts_mu", "ts_sigma", "mu_normal"]
```

Comment:
You aren't filling in a value for `mu_normal`. What is it for?

Reply:
my mistake, mixed up files